CodeGeneration/Sources/SyntaxSupport/KeywordSpec.swift

```diff
@@ -628,6 +632,8 @@ public enum Keyword: CaseIterable {
       return KeywordSpec("protocol", isLexerClassified: true)
     case .public:
       return KeywordSpec("public", isLexerClassified: true)
+    case .read:
+      return KeywordSpec("read")
```

Comment:
Should `read` also be behind an experimental feature, like `modify`?

Reply:
Thanks!  Fixed.